extension/src/panel.ts

```diff
@@ -26,28 +27,18 @@ interface EncodedSettings {
   base64ClientPrivateKey?: string
 }
 
-async function fileType(path: string): Promise<vscode.FileType | undefined> {
-  try {
-    const stat = await vscode.workspace.fs.stat(vscode.Uri.file(path))
-    return stat.type
-  } catch (err) {
-    // TODO: check exception and throw if unexpected
-    return undefined
-  }
-}
-
 export class HistoryDebuggerPanel {
   protected static _instance?: HistoryDebuggerPanel
+  protected static _server?: Server
 
-  static install(
-    extensionUri: vscode.Uri,
-    secretStorage: vscode.SecretStorage,
-    httpServerUrl: string,
-  ): HistoryDebuggerPanel {
-    if (this._instance === undefined) {
-      this._instance = new this(extensionUri, secretStorage, httpServerUrl)
+  static async install(extensionUri: vscode.Uri, secretStorage: vscode.SecretStorage): Promise<void> {
+    if (this._server === undefined) {
```

Comment:
The issue I pointed out is still not addressed here

Reply:
Ah I think I understand now. It needs to be assigned to the unresolved promise. Please advise if that is not in line with what you are thinking here and thanks for bearing with me.